rs/xnet/payload_builder/src/certified_slice_pool.rs

```diff
@@ -352,14 +355,12 @@ struct Payload {
 }
 
 /// Mean empty payload byte size: `LabelTree` with `"streams"` and `"header"`
-/// labels; subnet ID; and serialized header (3 variable length encoded
-/// integers).
-const EMPTY_PAYLOAD_BYTES: usize = 62;
+/// labels; subnet ID.
+const EMPTY_PAYLOAD_BYTES: usize = 49;
```

Comment:
Just from the running the failing test, it appeared to be off by 13. Seems like an odd number, but it works.

Reply:
Ideally, I'd run the test with various numbers here and do a binary search for the lower and upper bound; then pick the mean of the two numbers.

But it's potentially tedious work, so I can't ask that you do it, unless you actually want to.

Reply:
Well I checked the output for some examples and it was always 13.